docs/en/debugging/query-tracing.md

```diff
@@ -0,0 +1,170 @@
+# The OAP Query Tracing
+SkyWalking OAP provides the query tracing service to help users to debug and diagnose the query performance for the SkyWalking backend self.
+
+## Tracing Structure
+- Trace
+
+| Field     | Description                                     |
+|-----------|-------------------------------------------------|
+| traceId   | The unique ID of the trace                      |
+| condition | The query conditions                            |
+| startTime | The start time of the trace. In nanoseconds     |
+| endTime   | The end time of the trace. In nanoseconds       |
+| duration  | The operation name of the trace. in nanoseconds |
+| spans     | spans in this trace                             |
+
+- Span
+
+| Field        | Description                                                                                                      |
+|--------------|------------------------------------------------------------------------------------------------------------------|
+| spanId       | The unique ID of the span                                                                                        |
+| parentSpanId | The parent span ID of the span                                                                                   |
+| operation    | The operation name of the span                                                                                   |
+| startTime    | The start time of the span. In nanoseconds                                                                       |
+| endTime      | The end time of the span. In nanoseconds                                                                         |
+| duration     | The duration of the span. In nanoseconds                                                                         |
+| msg          | The message of the span, could include additional info such as request condition and response from the Database  |
+| error        | The error message of the span, if the span has an error.                                                         |
+
+
+## MQE Query Tracing
+MQE query tracing used to trace the query performance of the MQE query.
+
+### Debugging HTTP Service
```

Comment:
```suggestion
### Debugging through HTTP APIs
```

Reply:
We should add another part `Debugging with GraphQL bundled` to introduce the new flag and response in the metrics v3 query.